5-data-modeling/event-manifest-populator/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
We should remove what is on this page and link to [our docs website](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/loaders-storage-targets/events-manifest-populator/) instead (and update it if there is information that is on this page but not on docs).

Reply:
Thank you very much @benjben ! I will fix these in the "Update READMEs" commit, force push and check the docs for missing info there.
Concerning the submodule updates, i just did it manually.